modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc

```diff
@@ -0,0 +1,97 @@
+[[contact-methods-pushssh]]
+= SSH Push (with tunnel) contact method
+
+
+SSH Push (with tunnel) ([literal]``ssh-push-tunnel``) is used in environments where clients cannot reach the {productname} Server directly.
+In this environment, clients are located in a firewall-protected zone called a DMZ.
+No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
+
+This SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located on the DMZ.
+After all actions and events are executed, the tunnel is closed.
+
+The server uses SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
+
+
+// FiXME: only here with tunnel, otherwise move to contact-methods-saltssh.adoc
+[IMPORTANT]
+====
+Re-installing systems using the provisioning model is not currently supported on clients managed with SSH Push.
+====
+
+
+[IMPORTANT]
+====
+The tunnel is used to provide the access to the server through the encrypted tunnel.
+The repositories assigned to the SSH Push client (with tunnel) are provided through this tunnel only, thus it is not possible to use package manager tools directly from the client system because the repositories are available only while the tunnel is up.
+In other words, access is only possible if the session is initiated by the server.
+All package managing operations on the client can be performed from the server side only.
+====
+
+For tunneling connections via SSH, a port number is required for tunneling via HTTPS.
+The port number used by default is `1233`.
+To overwrite it, you can add a custom port numbers greater than 1024 to [path]``/etc/rhn/rhn.conf``:
+
+----
+ssh_push_port_https = high_port
+----
```

Comment:
Does it make sense to add note that services should be restarted after chaning this config parameter? At least the following should be restarted: `salt-secrets-config.service`, `tomcat.service`, `taskomatic.service`. Or maybe just refer to `spacewalk-service restart`.

Reply:
I think "yes".  Done.